### ISSUE TITLE
Popover autoClose not dynamic

### ISSUE THREAD

**Issue by tjw02** [ng-bootstrap/ng-bootstrap#4680]
Please remember, the issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
Hi, 

We have a need for the popover to have a dynamically working autoClose property. Currently looking at the ngbootstrap, the autoClose value is being used only during the opening of the popover. However, we have the use case where some action means that the popover can temporarily not be closed for the popover that is currently being displayed. An example use case is if some action inside the popover triggers a server call, and we want to keep the popover open with a small spinner in the corner until the server response is returned.

Cheers,
Tom.

### Link to minimally-working StackBlitz that reproduces the issue:

You can fork a StackBlitz from one of our [demos](https://ng-bootstrap.github.io/#/components) and use it as a starting point.

Please note that we can _not_ act on bug reports without a _minimal_ reproduction scenario in a StackBlitz. Here is why:
https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular:
17.2
ng-bootstrap:
16.0
Bootstrap:
5.3

**Comment by janwidmer** [comment]
@shyamal890 It seems to work, but you need to close the popover once after clicking on the button "Toggle AutoClose" in the case it was open, when you clicked "Toggle AutoClose"

**Comment by shyamal890** [comment]
Yes, I agree but that's the point once the popover is opened and if I change `auto_close` it should update how popover works. This shouldn't require closing and opening the popover again. I have already pushed a PR that solves this issue.
